torch/csrc/jit/mobile/import.cpp

```diff
@@ -303,6 +304,14 @@ void BytecodeDeserializer::parseFunctionSchema(
   }
 }
 
+void BytecodeDeserializer::init_upgrader(mobile::Function* function) {
+  for (auto const& [upgrader_name, upgrader_bytecode] : kUpgraderBytecode) {
```

Comment:
PyTorch OSS uses C++14, so unfortunately we need to avoid structured binding.

Reply:
Will be using kUpgradeFunction, so it's not needed any more